docs/tools/release.sh

```diff
@@ -44,27 +37,7 @@ then
     if [[ ! -z "${CLOUDFLARE_TOKEN}" ]]
     then
         sleep 1m
-        git diff --stat="9999,9999" --diff-filter=M HEAD~1 | grep '|' | awk '$1 ~ /\.html$/ { if ($3>6) { url="https://content.clickhouse.tech/"$1; sub(/index.html/, "", url); print "\""url"\""; }}' | split -l 25 /dev/stdin PURGE
-        for FILENAME in $(ls PURGE*)
-        do
-            POST_DATA=$(cat "${FILENAME}" | sed -n -e 'H;${x;s/\n/,/g;s/^,//;p;}' | awk '{print "{\"files\":["$0"]}";}')
-            sleep 3s
-            set +x
-            curl -X POST "https://api.cloudflare.com/client/v4/zones/4fc6fb1d46e87851605aa7fa69ca6fe0/purge_cache" -H "Authorization: Bearer ${CLOUDFLARE_TOKEN}" -H "Content-Type:application/json" --data "${POST_DATA}"
-            set -x
-            rm "${FILENAME}"
-        done
+        python3 purge-cache-for-changed-files.py
     fi
     cd "${BUILD_DIR}"
-    DOCKER_HASH=$(head -c 16 < /dev/urandom | xxd -p)
-fi
-
-QLOUD_ENDPOINT="https://platform.yandex-team.ru/api/v1"
-QLOUD_PROJECT="clickhouse.clickhouse-website"
-if [[ -z "$1" ]]
-then
-    QLOUD_ENV="${QLOUD_PROJECT}.test"
-else
-    QLOUD_ENV="${QLOUD_PROJECT}.prod"
 fi
-echo ">>> Successfully deployed ${TAG} ${DOCKER_HASH} to ${QLOUD_ENV} <<<"
```

Comment:
as far as I remember some of this output was used by the sandbox task, it might stop behaving well without proper adjustments

Reply:
It's ok, I'll fix them too.

Reply:
but let's try in this PR